_includes/progress.html

```diff
@@ -168,29 +138,32 @@
  * @return a Promise object that populates and creates DOM elements, or returns the status text on error
  */
 function retrieveReleases(url, linkBaseUrl) {
-    var releasesContainer = document.getElementById("repository-releases");
     return new Promise(function(resolve, reject) {
         const xhr = new XMLHttpRequest();
         xhr.open("GET", url);
         xhr.onload = function() {
             const releases = JSON.parse(this.response);
-            // we only want to iterate over 4 releases...
-            var min = Math.min(releases.length, 2);
-            releasesContainer.innerHTML = "";
+            var min = Math.min(releases.length, 15);
+            var foundCylc8Latest = false;
+            var foundCylc7Latest = false;
             for (var i = 0; i < min; ++i) {
                 var release = releases[i];
                 var name = release.name;
                 var publishedAt = release.published_at;
                 var body = release.body;
                 var link = linkBaseUrl + release.tag_name;
-                var releaseElement = createReleaseElement(name, link, publishedAt, body);
-                releasesContainer.appendChild(releaseElement);
-
-                // we also display the latest release metadata
-                if (i == 0) {
-                    populateLatestReleaseInformation(name, publishedAt);
-                }
-            }
+                // display latest release metadata
+                if (name.startsWith("cylc-7.8")) {
+                    populateLatestReleaseInformation("cylc7", name, link, publishedAt);
+                    foundCylc7Latest = true;
+                } else if (name.startsWith("cylc-flow-8")) {
+                    populateLatestReleaseInformation("cylc8", name, link, publishedAt);
+                    foundCylc8Latest = true;
+                };
+                if (foundCylc7Latest === true && foundCylc8Latest === true) {
```

Comment:
@kinow - I thought you might pull me up on this, because it looks retarded! I guess in this context - the values can only be boolean - it would be fine to just do `if (foundCylc7Latest && foundCylc8Latest)...`  (What's idiomatic JS for this?)

Reply:
(I know, I should know - embarrassing :grimacing: )

Reply:
I recently suggested removing a `if bool_var == True` in Python jupyterhub, but it is actually correct in that context to check for `== True`: FWIW here's the comment thread :smile:  https://github.com/jupyterhub/jupyterhub/pull/2739#discussion_r327505850

Not sure what's the idiomatic for JS. I'd say `(foundCylc7Latest && foundCylc8Latest)` should be better as long as we are sure both values are `boolean`.

Some people code always using the not-not operator in JS, `!!` https://stackoverflow.com/a/784946, e.g. in vue.js: https://github.com/vuejs/vue/blob/235064ebe19648420295f5ee57cf5ed5b72bbc16/src/core/observer/watcher.js#L59

So it's not that hard to find `==` or `=== true` in JS code, another example from JS https://github.com/vuejs/vue/blob/235064ebe19648420295f5ee57cf5ed5b72bbc16/src/core/util/env.js#L79

So definitely not embarassing :grimacing: I try to use JSDoc + tests to ensure we have the truthy values right, but we could have some bugs hidden in Cylc UI!
Before ES6

Reply:
Just wondering whether we can add some kind of *metadata* or *tag* to the milestone description that we can use here to identify the **next feature release** and/or the **next maintenance release**? If so, we can rewrite the above to be few of version number - so you don't end up having to change this again when Cylc 9 becomes the next feature release.

Reply:
@matthewrmshin - that's a good idea, but I'll punt to a future improvement, so that we can get this in now (it is better than the current situtation, and I have no time to go further right now.)